src/ex1/index.html

```diff
@@ -1,14 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <title>Exercise 1</title>
-    <link rel="stylesheet" href="style.css" />
-</head>
-<body>
+   <head>
+      <meta charset="UTF-8" />
+      <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```

Comment:
what is it and why do we need it?

Reply:
`<meta charset="UTF-8" />` - I learned that it is better to add it to every HTML page. I rechecked it now and it seems it is not necessary anymore so ill delete it:
https://dev.to/maggiecodes_/why-is-lt-meta-charset-utf-8-gt-important-59hl
https://stackoverflow.com/questions/14669352/is-the-charset-meta-tag-required-with-html5

Same thing for `<meta http-equiv="X-UA-Compatible" content="IE=edge" />`:
https://stackoverflow.com/questions/6771258/what-does-meta-http-equiv-x-ua-compatible-content-ie-edge-do
Will delete as well.